test/cli/0003.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+set -e
+
+name=$(basename $0)
+root="$(dirname $(dirname $(dirname $0)))"
+gomtree=$(readlink -f ${root}/gomtree)
+t=$(mktemp -d ${root}/go-mtree.XXXXXX)
```

Comment:
I'll switch it to `$TMPDIR`. We will automatically skip this test if `xattrs` don't work (line 9).